Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/RotationManipulators.cpp

```diff
@@ -158,4 +158,20 @@ namespace AzToolsFramework
     {
         m_circleBoundWidth = circleBoundWidth;
     }
+
+    void RotationManipulators::VisitManipulators(const ManipulatorVisitCallback&& callback)
+    {
+        for (auto& manipulator : m_localAngularManipulators)
+        {
+            if (manipulator)
+            {
+                callback(manipulator);
+            }
+        }
+        if (m_viewAngularManipulator)
+        {
+            callback(m_viewAngularManipulator);
+        }
+    }
+
```

Comment:
I'm really sorry but we don't need this, it already exists in the form of `ProcessManipulators` (e.g `RotationManipulators::ProcessManipulators` and `ScaleManipulators::ProcessManipulators` etc.. - please remove these calls and take a look at my other comment

Reply:
do you want to make ProcessManipulators public?

Reply:
done

Reply:
Perfect thanks! 👍